doc/concepts/interconnectivity.rst

```diff
@@ -39,6 +39,17 @@ A recipe lets you define which sites are connected to which.
    action potential
       Spikes travel over :term:`connections <connection>`. In a synapse, they generate an event.
 
+   spike detector
+   Placed on a cell. Possible source of a connection. Detects crossing of a fixed threshold and generates
+   corresponding events.
+
+   spike source cell
```

Comment:
Same: have a `:ref:` to Python API `arbor.spike_source_cell`. It may be confusing to define the concept without referencing the class.